_docs/setup/kubernetes/quick-start-gke-dm.md

```diff
@@ -0,0 +1,230 @@
+---
+title: Quick Start with Google Kubernetes Engine
+overview: Quick Start instructions to setup the Istio service using Google Kubernetes Engine (GKE)
+
+order: 11
+
+layout: docs
+type: markdown
+---
+
+{% include home.html %}
+
+
+Quick Start instructions to install and run Istio in [Google Kubernetes Engine](https://cloud.google.com/kubernetes-engine/){:target="_blank"} (GKE) using [Google Cloud Deployment Manager](https://cloud.google.com/deployment-manager/){:target="_blank"}.
+
+This Quick Start creates a new GKE cluster, installs Istio and then deploys the [BookInfo](https://istio.io/docs/guides/bookinfo.html){:target="_blank"} sample application.  It uses Deployment Manager to automate the steps detailed in the [Istio on Kubernetes setup guide](/docs/setup/kubernetes/quick-start.html){:target="_blank"} for Kubernetes Engine
+
+> **Note:** The default install will create a GKE [**alpha** cluster](https://cloud.google.com/kubernetes-engine/docs/alpha-clusters){:target="_blank"} that allows for [automatic sidecar injection](https://istio.io/docs/setup/kubernetes/sidecar-injection.html#automatic-sidecar-injection){:target="_blank"}.  As it is an alpha cluster, it does not support automatic node or master upgrades and will be automatically deleted after 30 days.
+
+## Prerequisites
+
+This sample requires a valid Google Cloud Platform project with billing enabled.  
+- If you are not an existing GCP user, you may be able to enroll for a $300 US [Free Trial](https://cloud.google.com/free/){:target="_blank"} credit.
+
+- You must install and configure [gcloud command line tool](https://cloud.google.com/sdk/docs/){:target="_blank"} and include the ```kubectl``` component (```gcloud components install kubectl```).  If you do not wish to install the gcloud client, you can use [Google Cloud Shell](https://cloud.google.com/shell/docs/){:target="_blank"} to perform these tasks.
+
+- <img src="{{home}}/img/exclamation-mark.svg" alt="Warning" title="Warning" style="width: 32px; display:inline" /> You must set your default compute service account to include:
```

Comment:
This doesn't need the warning bit, it's just a prerequisite like the others: just start the bullet with "Your default compute service account must include the following IAM roles:"

Reply:
I'd really like to leave this in;  I do know this is labeled but w/o this setting, the deployment fails /w very little in the ways of error messages (deployment manager doesn't surface in-script errors).  I'd like to highlight this very clearly for users.